inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/AuditServiceImpl.java

```diff
@@ -205,20 +205,21 @@ public List<AuditVO> listByCondition(AuditRequest request) throws Exception {
                 String format = "%Y-%m-%d %H:%i:00";
                 // Support min agg at now
                 DateTimeFormatter forPattern = DateTimeFormat.forPattern("yyyy-MM-dd");
-                DateTime dtDate = forPattern.parseDateTime(request.getDt());
-                String eDate = dtDate.plusDays(1).toString(forPattern);
+                DateTime edtDate = forPattern.parseDateTime(request.getEndDate());
+                String endDate = edtDate.plusDays(1).toString(forPattern);
```

Comment:
May change to `DateTime endDate` and `String endDateStr` is better.

Reply:
Done.